apis/python/tests/test_query_condition.py

```diff
@@ -243,6 +243,11 @@ def test_eval_error_conditions(malformed_condition):
             "louvain == leuko-cyte",
             "Unable to parse expression component.* did you mean to quote it as a string",
         ],
+        # FOO
```

Comment:
What's the meaning of this line?

Reply:
dangit. Thanks @ryan-williams for the catch.

Reply:
#2875